test/known_issues/test-vm-inherited_properties.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+// Ref: https://github.com/nodejs/node/issues/5350
+
+require('../common');
+const vm = require('vm');
+const assert = require('assert');
+
+const base = {
+  prop_base: 1,
+};
+
+const sandbox = Object.create(base, {
+  prop_sandbox: {value: 3}
+});
+
+const context = vm.createContext(sandbox);
+
+const result = vm.runInContext("this.hasOwnProperty('prop_base');", context);
```

Comment:
Ok. I asked for future reference.  
I used template strings here: https://github.com/nodejs/node/pull/10272/files
and looks like the linter was ok with it